### ISSUE TITLE
Check param count

### ISSUE THREAD

**Issue by voldemortX** [lucastabelini/LaneATT#79]
@lucastabelini Hi! I've tested the parameter count with `speed.py`, but I found resnet122 has less than half parameters compared to resnet34, it seems the lambda layer (padding) is not counted, but I think it does not contain any parameters. Can you validate this for me?

My results (tusimple ymls):

resnet18:
MACs: 9.335G
Params: 12.019M

resnet34:
MACs: 18.005G
Params: 22.127M (same as in the paper)

resnet122: 
MACs: 70.471G
Params: **8.545M**

**Comment by lucastabelini** [comment]
That is correct. The ResNet-122 is from a different "family" of ResNets. Its convolutional layers have a smaller number of filters. Therefore, even though the number of layers is larger, the number of parameters can be smaller. It is described in the original ResNet paper in their CIFAR-10 experiments.

**Comment by voldemortX** [comment]
Thanks!

Could I squeeze in a small unrelated question here.  How did you get the ultra fast lane detection resnet34 results in your paper? I couldn't find its tusimple full results (fp fn etc.) anywhere. If you still remember this.

**Comment by lucastabelini** [comment]
I had to run their evaluation script on the given pretrained model. Those results are surprisingly not shown in their paper. However, since the only pretrained model they provide is for the ResNet-18 backbone, I had to train the other one (which uses ResNet-34).

**Comment by voldemortX** [comment]
Thanks a lot!
